src/Transition/OutputRulesFiles/OutputChanges9-3-0-to-9-4-0.md

```diff
@@ -159,13 +159,21 @@ See [8121](https://github.com/NREL/EnergyPlus/pull/8121)
 
 (a) From "FuelOil#1" in 9.3 to "FuelOilNo1" in 9.4
 (b) From "FuelOil#2" in 9.3 to "FuelOilNo2" in 9.4
-
- 
 (c) From "Fuel Oil #1" in 9.3 to "FuelOilNo1" in 9.4
 (d) From "Fuel Oil #2" in 9.3 to "FuelOilNo2" in 9.4
 
 ### table output headings moving from:
  (a) "Fuel Oil #1" in 9.3 to "Fuel Oil No 1" in 9.4
  (b) "Fuel Oil #2" in 9.3 to "Fuel Oil No 2" in 9.4
  
-
+### EnergyManagementSystem:Actuator Actuated Component Control Type
```

Comment:
Not sure we need to repeat this in the output changes doc.

Reply:
I removed this repeated block from the output changes doc.